src/lucky_record/exceptions.cr

```diff
@@ -0,0 +1,9 @@
+module LuckyRecord
+  module Exceptions
+    class Base < Exception
+    end
+
+    class InvalidForm < Base
```

Comment:
I kinda guessed on an appropriate class name here.

Reply:
I think InvalidForm is a good name. It can go here with the other errors: https://github.com/luckyframework/record/blob/master/src/lucky_record/errors.cr

Also need `Error` at the end. Might be worth passing in the form name and form object as well? That way it can be used in the output or for automatically handling form errors in JSON APIs